READ0105.txt

```diff
@@ -23,4 +23,4 @@ https://docs.google.com/document/d/1SipNvEDdPKj-y0f8mU8-vbwYyOUtohnCZMwos2kFEaI/
 
 
 
-Hi did you buy the gift?
```

Comment:
Hmm, That was very nice experience, I asked to your client what she want.

Reply:
So, what did she want?

Reply:
I think the client require red money pocket. haha

Reply:
:O

Reply:
That's good

Reply:
Hmm.